Tests/Runtime/Core/BucketManagement/BucketStoreTests.cs

```diff
@@ -545,6 +578,7 @@ public void PutLifecycleConfiguration_WhenInvalidBucketPolicyPassed_IsNotSuccess
             Assert.AreEqual(putLifecycleConfigurationResponse.ErrorCode, ErrorCode.InvalidBucketPolicy);
         }
 
+        //TODO: Fix this test case. Currently failing since BucketPolicy not passed == BucketPolicy.None which is considered valid
```

Comment:
This test case was for an older requirement and can be removed

Reply:
sounds good, will remove